### ISSUE TITLE
Problem importing sys

### ISSUE THREAD

**Issue by pkhoueiry** [microsoft/BioGPT#40]
Running "KD-DTI" example results in error: src module not found. "BioGPT" directory needs to be added to the path...

**Comment by renqianluo** [comment]
Yes, if you run it from path other than the root path of BioGPT, you should add it to the PYTHONPATH
